samples/nrf_desktop/src/modules/config.c

```diff
@@ -18,16 +18,21 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_CONFIG_LOG_LEVEL);
 
-#define KEY_LEN 9
+#define KEY_LEN 11
 
 /* Array of pointers to data loaded from settings */
-static struct config_event *loaded_data[CONFIG_EVENT_ID_MAX];
+#define LOADED_DATA_MAX 4
```

Comment:
What about using `SENSOR_OPT_COUNT ` from config_event.h or even better, some macro that will sum up the possible options?

Reply:
Yes, for now we can do this but truly I would like to move the module specific data store into module file as discussed offline.

Reply:
Fixed